internal/provider/resource_eva_role.go

```diff
@@ -60,11 +60,19 @@ func (t roleProviderType) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Dia
 						},
 						"requires_elevation": {
 							MarkdownDescription: "whether functionality requires elevation or not",
-							Required:            true,
+							Optional:            true,
+							Computed:            true,
```

Comment:
How come its computed? Because a default is set?

Reply:
Yeah, once you have modifiers, you need to mark `computed` as `true`